<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>350</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Цвет закраски и фона</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>331</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Цвет закраски (RGB)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="RedFore">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="GreenFore">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="BlueFore">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>331</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>    Цвет фона (RGB)   </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="RedBack">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="GreenBack">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="BlueBack">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="saveColorButton">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 12px;
color: white;
background-color: rgb( 75, 183, 141 );

</string>
     </property>
     <property name="text">
      <string>Сохранить всё</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveDrawButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 12px;
color: white;
background-color: rgb( 75, 183, 141 );

</string>
     </property>
     <property name="text">
      <string>Сохранить 
цвет закраски</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="quiteProgramButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>500</y>
      <width>101</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
color: white;
background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Завершить 
работу</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>350</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Добавление точки</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>321</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Координата по Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="appendY"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="appendButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 12px;
color: white;
background-color: rgb( 75, 183, 141 );

</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Координата по X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="appendX"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>620</width>
      <height>580</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="inputRadioButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Режим ввода многоугольника</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="inputCurveRadioButton">
       <property name="text">
        <string>Режим ввода произвольной области</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="fillRadioButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Режим заливки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="FillDelayRadioButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Режим заливки с задержкой</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="endInputButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>101</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
color: white;
background-color: rgb( 75, 183, 141 );
</string>
    </property>
    <property name="text">
     <string>Закончить ввод</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>500</y>
      <width>101</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 12px;
color: white;
background-color: rgb( 75, 183, 141 );
</string>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quiteProgramButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>951</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>997</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
